src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -276,8 +292,8 @@ const styles = StyleSheet.create({
     justifyContent: 'center'
   },
   newPostText: {
-    color: COLORS.holyTosca,
-    marginRight: 11,
+    color: COLORS.holytosca,
+    marginRight: dimen.normalizeDimen(11),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Using the `dimen.normalizeDimen` function to set style values is a good practice for maintaining consistent layout and sizing across different screen sizes.
```diff
- color: COLORS.holyTosca,
- marginRight: 11,
+ color: COLORS.holytosca,
+ marginRight: dimen.normalizeDimen(11),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->